docs/guides/blocks-filer-block-attributes-override.md

```diff
@@ -0,0 +1,49 @@
+---
+id: blocks-filter-block-attributes-override
+title: Filter Attributes Override
+---
+
+This filter gives you and ability to hook you changes to blocks/wrapper attributes after it is registred in editor but before it is rendered in React.
+
+Usage of this method is vast. You can for example use it to provide differend blocks/wrapper defaults depending on the post type. For example you have a button block that you want to be default color red but on post type `post` you want that button to be black by default. Using this hook you can easily do this.
+
+> This method can only change the default values of the attributes. Once the value is saved in the database and it is different from the default it will stay in the detabase and all this changes will not apply to it.
```

Comment:
```suggestion
> This method can only change the default values of the attributes. Once the value is saved in the database and it is different from the default it will stay in the database and all the changes won't apply to it.
```

Reply:
👍